flutter/lib/src/integrations/on_error_integration.dart

```diff
@@ -42,7 +42,7 @@ class OnErrorIntegration implements Integration<SentryFlutterOptions> {
     _defaultOnError = wrapper.onError;
 
     _integrationOnError = (Object exception, StackTrace stackTrace) {
-      final handled = _defaultOnError?.call(exception, stackTrace) ?? true;
+      final handled = _defaultOnError?.call(exception, stackTrace) ?? false;
```

Comment:
I'm not sure if we should use the result of this function as the sentry `handled` flag.
I believe we should just use `false` since the concept is different.
About fallback to `true` or `false`, I'd check whats the default value for `https://master-api.flutter.dev/flutter/dart-ui/ErrorCallback.html since we have to return something but `_defaultOnError` can be null.
@ueman opinions?

Reply:
If a user provides their own handler through `_defaultOnError', shouldn't we interpret it as 'handled'? 🤔

Reply:
Nope, I've got nothing to add to what you said.

Reply:
Same as https://github.com/getsentry/sentry-dart/pull/1186#discussion_r1047052173
I'd still treat it as `handled: false`, this is a Sentry flag and not related to Flutters forwarding error handling or not.
We just forward the call to `_defaultOnError` as a good citizen.
If the callback has to return the fallback of `true` or `false`, I am unsure, I'd dig into the default `ErrorCallback` in the Flutter framework and use what they use as a default value.

Reply:
I was not able to find and ocurence in `flutter` or `flutter/engine` where this callback is assigned a default implementation. Maybe i'm searching in the wrong place. The documentation mentioned that if false is returned, the error will be [printed to sdtout](https://master-api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html). So i'd suggest to return the value we had previously and treat the mechanisms handled separately (always set to false).

Reply:
The relevant code is here I believe https://github.com/flutter/engine/blob/74a9c7e0f9b16cf8e26b2d1ce7b7777ae2a07572/lib/ui/platform_dispatcher.dart#L1141-L1183

Reply:
Yep, return false as fallback